coursemapper-kg/recommendation/app/services/course_materials/db/neo4_db.py

```
@@ -1704,6 +1784,948 @@
+                user_id=user_id,
+                cid=cid,
+                weight=weight,
+                mid=mid,
+                status=status
+            ).single()
+
+            r_detail = { "cm_id": r_detail["cm_id"], "cid": r_detail["cid"], 
+                        "weight": r_detail["weight"], "mid": r_detail["mid"], 
+                        "status": r_detail["status"]
+                    }
+            return r_detail
```

Ruff (0.12.2)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)